docs/advanced-configuration-options.md

```diff
@@ -18,6 +18,14 @@ which could have unknown side-effects.
 | ------------------------------ | ------------------------------ | -------------------------------------------------------------------------------------------------- |
 | otel.javaagent.exclude-classes | OTEL_JAVAAGENT_EXCLUDE_CLASSES | Suppresses all instrumentation for specific classes, format is "my.package.MyClass,my.package2.\*" |
 
+## Excluding specific classes loaders
+
+This can be used to Ignore the class loader specified and exclude it from being instrumented.
```

Comment:
> This can be used to Ignore the class loader specified and exclude it from being instrumented.

This is not completely true. This flag does not prevent instrumenting the class loader, it prevents instrumenting classes defined by the given loader. Perhaps `This option can be used to exclude classes loaded by given class loaders from being instrumented.`